tests/test_requests.py

```diff
@@ -1353,6 +1353,13 @@ def test_response_json_when_content_is_None(self, httpbin):
         with pytest.raises(ValueError):
             r.json()
 
+    def test_response_without_release_conn(self):
+        """Test `close` call for non-urllib3-like raw objects.
+        Should work when `release_conn` attr doesn't exist on `response.raw`.
+        """
+        resp = requests.Response()
+        resp.raw = StringIO.StringIO('test')
+        resp.close()
```

Comment:
Can we make some kind of positive assertion here? I don't want us to get to a place where we fix this by having `close` be a no-op without this test failing. =)